src/BuildConfig.cc

```diff
@@ -64,6 +64,7 @@ operator<<(std::ostream& os, VarType type) {
 
 BuildConfig::BuildConfig(const std::string& packageName, const bool isDebug)
     : packageName{ packageName }, isDebug{ isDebug } {
```

Comment:
Now this line looks awkward. Can you also please move this to the constructor body?

```suggestion
BuildConfig::BuildConfig(const std::string& packageName, const bool isDebug) {
    packageName = packageName;
    isDebug = isDebug; 
```

Reply:
Clang tidy throws an error if I do that.

Reply:
Oops, sorry!